openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml

```
@@ -1,11 +1,13 @@
 type: object
-title: Full Payment Card Token
-allOf:
-  - $ref: ../Common/CommonPaymentToken.yaml
-required:
+title: Payment Card Token
+allOf: 
+  - $ref: ./CompositeToken.yaml
+description: |
+  Provide billingAddress in order to generate a Full Payment Card Token.
+  If billingAddress is not provided the token can not be used for payments.
```

issue: Missing detail

I think we are missing detail on what the operation does (I'm not 100% sure that my suggestion is correct)

```suggestion
  Creates a payment card token. A payment card token is a string that 
  represents a customer's payment card details. It can be used once and expires within 24 hours.
  To generate a full payment card token, you must provide the `billingAddress`.
  If `billingAddress` is not provided, the token can not be used for payments.
```

---

I belive it is first use or 30 minutes. See description [here](https://api-reference.rebilly.com/tag/Payment-Tokens/operation/PostToken)

---

I updated my suggestion based on your input